base/deprecated.jl

```diff
@@ -378,14 +378,8 @@ helping for type inference.
     In Julia 1.8 and higher, it is favorable to use [`@assume_effects`](@ref) instead of `@pure`.
     This is because `@assume_effects` allows a finer grained control over Julia's purity
     modeling and the effect system enables a wider range of optimizations.
-
-!!! note
```

Comment:
We can still have the note saying to prefer the new style maybe?

Reply:
Something more than the sentence just above?

Reply:
Yeah you're right, it basically says that there.